src/compute-client/src/controller/instance.rs

```diff
@@ -650,6 +664,14 @@ impl<T: ComputeControllerTimestamp> Instance<T> {
         all_caught_up
     }
 
+    /// Returns `true` iff all collections are hydrated on at least one replica.
+    ///
+    /// This also returns `true` in case this cluster does not have any
+    /// replicas.
+    pub fn all_collections_hydrated(&self) -> bool {
+        self.all_collections_hydrated_filtered(None)
+    }
```

Comment:
I'm not confident this is a good way to handle this what is essentially an overloaded method.